src/main/resources/swagger/api-docs.yaml

```diff
@@ -204,6 +318,27 @@ paths:
           description: Method Repository configuration detail
         500:
           description: Internal Server Error
+    delete:
+      tags:
+        - Method Repository
+      summary: Redact a single configuration.
+      description: |
+        Redacts a configuration and all of its associated configurations
+      parameters:
+        - $ref: '#/parameters/configNamespaceParam'
+        - $ref: '#/parameters/configNameParam'
+        - $ref: '#/parameters/configSnapshotId'
+      responses:
+        200:
+          description: >
+            An integer showing how many configurations were deleted. Should be
```

Comment:
They're technically different - folded style vs. literal style; see http://www.yaml.org/spec/1.2/spec.html#id2796251.  This is all copy/paste from agora's yaml, so I left it as-is.